.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 jobs:
   frontend:
     docker:
-      - image: circleci/node:14.17
+      - image: cimg/node:14.20
```

Comment:
The latest NodeJS version is `18.7.0` and the LTS Version is `16.16.0`. Might be worth an update.

Reply:
I updated it to the LTS.

Reply:
Uh now the build complains about the legacy peer dependencies.